src/Wrapper.js

```diff
@@ -234,7 +234,7 @@ class Wrapper extends React.PureComponent {
                 connected={connected}
                 notifications={notifications.length}
                 daoAddress={daoAddress}
-                openProgress={progress}
+                swipeProgress={progress}
                 autoClosing={autoClosingPanel}
```

Comment:
Wondering if we can roll `autoClosingPanel` and `menuSwipeEnabled` into the same prop, perhaps `collapsibleMenuBar`?

Reply:
Agreed, especially since the swipe feature doesn’t only depend on the viewport’s width.